.storybook/theme/ThemeProvider.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  CSSReset,
+  ThemeProvider as ChakraThemeProvider,
+  theme as chakraDefaultTheme,
+} from '@chakra-ui/core';
+
+// @TODO: move this into src/components
+// @TODO: speak with designers about the colors palette
+const theme = {
+  ...chakraDefaultTheme,
+  breakpoints: ['28.75rem', '50rem', '68.75rem'],
+  colors: {
+    ...chakraDefaultTheme.colors,
+    primary: {
+      400: '#ff4d95',
+      500: '#ff2773',
+      600: '#ff4d95', // hover
```

Comment:
Is this how Chakra uses colors? Can't we keep it as we have it right now? `primary.whatever`

Reply:
Technically we can, but for example, Buttons will use `{color}.500` for background, `{color}.600` for hover, `{color}.400` for active, etc.
The idea is to have a wider palette with more explicit names, rather than `{color}-light|-light2` etc.
It's a point I need to talk to with the design team